site/public/js/gradeable-notebook.js

```diff
@@ -94,43 +128,65 @@ function saveNotebookToLocal() {
+                found = true;
+                //check off proper inputs
+                for (let i = 0; i < answers.length; i++) {
+                    if ($(this)[0].defaultValue === answers[i]) {
+                        $(this).prop('checked', true);
+                    }
+                }
             });
+            if (!found) {
+                not_found.push(inputs.multiple_choice[id]);
+            }
         }
-        // Next, we restore short-answer boxes
-        for (const id in state.short_answer) {
-            const answer = state.short_answer[id][0];
-            // Restore the answer and trigger change events (for setting button states)
-            // (see https://stackoverflow.com/questions/4672505/why-does-the-jquery-change-event-not-trigger-when-i-set-the-value-of-a-select-us)
-            $(`#${id}`).val(answer).trigger('input');
+
+        //restore short answers
+        for (const id in inputs.codebox) {
+            const {file_name, value} = inputs.codebox[id];
+            const question = $(`.short_answer > div[data-filename="${file_name}"]`);
+            //fill in proper values if question is found
+            if (question.length > 0) {
+                const editor = question[0].querySelector('.CodeMirror').CodeMirror;
+                editor.setValue(value);
+            }
+            else {
+                not_found.push(inputs.codebox[id][0]);
+            }
         }
-        // Finally, we restore codeboxes
-        for (const id in state.codebox) {
-            const answer = state.codebox[id][0];
-            const codebox = $(`#${id} .CodeMirror`).get(0);
-            // If this box no longer exists, then don't attempt to update the
-            // answer. The autosave data is probably for an older version of
-            // the gradeable at this point, see issue #5351.
-            if (!codebox) {
-                continue;
+
+        //if there are answers that could not be placed anywhere
+        if (not_found.length > 0) {
+            const old_answers_div = document.createElement('div');
+            old_answers_div.id = 'old-answers';
+            old_answers_div.classList.add ('box', 'red-background');
+
+            //create header text to warn user
+            const old_answers_header = document.createElement('h4');
+            old_answers_header.innerHTML = 'Answer(s) could not be restored you will have to copy and paste them in the proper place';
+            //add header to container
+            old_answers_div.appendChild(old_answers_header);
+
+            // for ... of loop loops through all elements in array, while for ... in loop loops through
+            // all indexes/keys in arrays/objects
+            for (const i in not_found) {
+                const answer_text = document.createElement('p');
+                answer_text.innerHTML = not_found[i];
+                old_answers_div.appendChild(answer_text);
             }
```

Comment:
```suggestion
            old_answers_div.classList.add('box');
            old_answers_div.style.backgroundColor = 'var(--alert-background-danger-red)';

            //create header text to warn user
            const old_answers_header = document.createElement('h4');
            old_answers_header.style.color = 'var(--focus-danger-red)';
            old_answers_header.innerHTML = 'Answer(s) could not be restored you will have to copy and paste them in the proper place';
            //add header to container
            old_answers_div.appendChild(old_answers_header);

            // for ... of loop loops through all elements in array, while for ... in loop loops through
            // all indexes/keys in arrays/objects
            const old_answers_list = document.createElement('ol');
            old_answers_list.style.marginLeft = '3em';
            for (const i in not_found) {
                const answer_text = document.createElement('li');
                answer_text.innerHTML = not_found[i];
                old_answers_list.appendChild(answer_text);
            }
            old_answers_div.appendChild(old_answers_list);
```
Upon actually testing the code I gave you, I realized it is very ugly. Try this instead.